src/Tizen.TV.Extension.UIControls.Forms/DRMMediaSource.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Tizen.TV.UIControls.Forms;
+
+namespace Tizen.TV.Extension.UIControls.Forms
+{
+    public enum DRMType
+    {
+        Playready,
+        //Verimatrix,
+        //WidevineModular
+    }
+
+    public class DRMMediaSource : MediaSource
+    {
+
+        public DRMType DRMType { get; set; }
+        public Dictionary<string, object> Extradata { get; set; }
```

Comment:
Why is the value type an object? It must be not support all object types, you need to provide more specific types